CHANGELOG.md

```diff
@@ -2,6 +2,35 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+## Mapbox Navigation SDK 2.0.0-beta.8 - April 30, 2021
+
+ For details on how v2 differs from v1 and guidance on migrating from v1 of the Mapbox Navigation SDK for Android to the v2 public preview, see [2.0 Navigation SDK Migration Guide](https://github.com/mapbox/mapbox-navigation-android/wiki/2.0-Navigation-SDK-Migration-Guide).
+
+ ### Changelog
+ #### Features
+ * Customize voice guidance player. [#4278](https://github.com/mapbox/mapbox-navigation-android/pull/4278)
+ * Introduce `NavigationCameraTransitionOptions`. [#4332](https://github.com/mapbox/mapbox-navigation-android/pull/4332)
+ #### Bug fixes and improvements
+ * Enable native crashlytics in robo tests. [#4306](https://github.com/mapbox/mapbox-navigation-android/pull/4306)
+ * Refactor camera padding in example `MapboxNavigationActivity`. [#4317](https://github.com/mapbox/mapbox-navigation-android/pull/4317)
+ * Make route refresh interval configurable. [#4321](https://github.com/mapbox/mapbox-navigation-android/pull/4321)
+ * Remove `@JvmOverloads` from all Mapbox Views. [#4323](https://github.com/mapbox/mapbox-navigation-android/pull/4323)
+ * Add ktlint check via Makefile to `qa-test-app` module. [#4324](https://github.com/mapbox/mapbox-navigation-android/pull/4324)
+ * Refactor building selection in favor to expressions rather than source layers. [#4327](https://github.com/mapbox/mapbox-navigation-android/pull/4327)
+ * Change the positioning of `MapboxNavigationViewportDataSourceDebugger` layer on the map. This layer will now be positioned on top of the map and will not be a part of the map style. [#4334](https://github.com/mapbox/mapbox-navigation-android/pull/4334)
+ * Fixed replay's simulated driver issue where it will nearly stop on the freeway, by normalizing bearing for speed profile calculations. [#4338](https://github.com/mapbox/mapbox-navigation-android/pull/4338)
+ * Improve the behavior of puck in free drive by keeping the user location at the bottom edge of padding rather than center of the screen. [#4343](https://github.com/mapbox/mapbox-navigation-android/pull/4343)
```

Comment:
This doesn't contain the full message from the changelog clause in the PR. Especially misses the breaking change notification.